tests/compute/test_nccl.py

```diff
@@ -52,6 +52,36 @@ def test_nccl_sparse_pull_single():
     exp_rv = F.gather_row(value, req_index)
     assert F.array_equal(rv, exp_rv)
 
+@unittest.skipIf(F._default_context_str == 'cpu', reason="NCCL only runs on GPU.")
```

Comment:
Do we need to skip windows?

Reply:
Do we not enable NCCL on windows builds? Or is there another reason this should skip it?